scaffolding-chef-infra/lib/windows/bootstrap-chunk.rb

```diff
@@ -1 +1 @@
-ENV['PATH'] += ";C:/WINDOWS;C:/WINDOWS/system32/;C:/WINDOWS/system32/WindowsPowerShell/v1.0;C:/ProgramData/chocolatey/bin"
+ENV['PATH'] += ";C:/WINDOWS;C:/WINDOWS/system32/;C:/WINDOWS/system32/WindowsPowerShell/v1.0;C:/ProgramData/chocolatey/bin;C:/Habitat/;C:/ProgramData/Habitat"
```

Comment:
I'm not sure I understand or agree with this change. I think that if habitat needs to be here it should already be on `PATH` as we are adding to it.